tests/testStreamOpen.cpp

```diff
@@ -25,8 +25,11 @@ class StreamOpen : public ::testing::Test {
 protected:
 
     bool openStream(){
+        mBuilder.setAudioApi(AudioApi::OpenSLES);
```

Comment:
If the whole file uses OpenSL ES then the file name should reflect that. Otherwise just do it in specific tests.

Or pass ApiIndex in openStream call.

Reply:
> Or pass ApiIndex in openStream call.

Done